aziotctl/aziotctl-common/src/system/mod.rs

```diff
@@ -37,6 +38,28 @@ pub const SERVICE_DEFINITIONS: &[&ServiceDefinition] = &[
     },
 ];
 
+// Ordering is not important here, we defer it to snapd
+#[cfg(feature = "snapctl")]
+pub const SERVICE_DEFINITIONS: &[&ServiceDefinition] = &[
+    &ServiceDefinition {
+        service: "identityd",
```

Comment:
@alexclewontin I was looking into why the `system logs` command for iotedge and aziotctl is always empty output.  This service name gets used in the aztioctl-common/src/system/system_logs.rs when it shells out to `journalctl -u <name>`.  I noticed that the names for services in a snap end up being `snap.<package>.<service>`.  Does that sound right to you and is that durable or would it be considered an implementation detail for snaps on which the code cannot depend?  I tried a local change with those service names and found the `system logs` command now worked as expected.  

Iterating... I'll have some proposed changes later.